src/main/java/com/edhou/songr/controllers/AlbumController.java

```diff
@@ -0,0 +1,65 @@
+package com.edhou.songr.controllers;
+
+import com.edhou.songr.models.Album;
+import com.edhou.songr.models.AlbumRepository;
+import com.edhou.songr.services.FileUploadService;
+import com.google.gson.Gson;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.util.StringUtils;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.servlet.view.RedirectView;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.List;
+
+@Controller
+public class AlbumController {
+    static final String ALBUMS_PATH = "src/main/resources/albums.json";
+    Gson gson = new Gson();
+    @Autowired
+    public AlbumRepository albumRepository;
+    @Autowired
```

Comment:
This tells Spring to inject our annotated dependency class FileUploadService into this class.